api/api/util.py

```diff
@@ -154,6 +154,39 @@ def remove_nones_to_dict(dct):
             for k, v in dct.items() if v is not None}
 
 
+def parse_api_param(param: [str, None], param_type: str) -> [typing.Dict, None]:
+    """Parses an str parameter from the API query and returns a dictionary the framework can process
+
+    :param param: Str parameter coming from the API.
+    :param param_type: Type of parameter -> search or sort
+    :return: A dictionary
+    """
+    if param is not None:
+        return parse_search_param(param) if param_type == 'search' else parse_sort_param(param)
+    else:
+        return None
```

Comment:
I would change this wraper into something like:
```
my_func = f'_parse_{param_type}'
parser = globals().get(my_func, default=lambda x: param)
return parser(param)
```

Reply:
Done in ec0e3e9cff595db5b614e0050c0dd7d7288b6336